tst/plistqueue.tst

```diff
@@ -0,0 +1,79 @@
+gap> q := PlistQueue();
+<queue with 0/64 entries>
+
+#
+gap> PushFront(q, 15);
+gap> PopFront(q);
+15
+gap> PushBack(q, 15);
+gap> PopBack(q);
+15
+gap> Push(q, 15);
+gap> Pop(q);
+15
+gap> IsEmpty(q);
+true
+
+# test size, make sure it is bigger than
+# initial capacity so that expansion happens
+gap> N:=67;;
```

Comment:
Why just 67? That seems to rely on a queue having a initial capacity of 64... So at the very least, I'd make this something like `Capacity(q) + 3`, but probably better to just use a big value, like 1000?